src/timeline/Timeline.tsx

```diff
@@ -39,11 +61,14 @@ export default class Timeline extends React.PureComponent {
     events: [],
     format24h: true
   };
+  calendarHeight: number;
+  private _scrollView: any;
```

Comment:
_scrollView is a ref type not any. Also please use React.createRef and fix its usages.